### DISCUSSION TITLE
Adding "jsonls"

### DISCUSSION THREAD

**Discussion by kay-adamof** [mason-org/mason.nvim#502] [API suggestions]
There is a ["jsonls" language server](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#jsonls) but it's not found in mason installer.

Can I add it and make pull request?

**Comment by williamboman** [comment]
See `json-lsp`!

**Reply by kay-adamof** [reply]
Sorry, it was my mistake.
Thank you for replying me.
